dallo/src/ops.rs

```diff
@@ -14,6 +14,6 @@ mod ext {
     }
 }
 
-pub fn self_id() -> &'static ModuleId {
-    unsafe { &ext::SELF_ID }
+pub fn self_id() -> ModuleId {
```

Comment:
I'm not sure about this. Why would we copy `self_id` onto the contract's stack?

Reply:
Resolving as per our out of band discussion. (The compiler will most likely remove any unnecessary copy anyway)